backend/src/aggregator.rs

```diff
@@ -165,6 +185,34 @@ impl Handler<Subscribe> for Aggregator {
     }
 }
 
+impl Handler<SendFinality> for Aggregator {
+    type Result = bool;
```

Comment:
this will turn from `bool` to `()`, changes the `handle` method accordingly since we don't care about the result.